resources/waterheater.rb

```diff
@@ -575,15 +579,26 @@ def self.apply_heatpump(model, runner, space, weather, t_set, vol, ef,
   end
 
   def self.apply_indirect(model, runner, space, cap, vol, t_set, oncycle_p, offcycle_p, ec_adj,
-                          nbeds, boiler, boiler_plant_loop, boiler_fuel_type, dhw_map, sys_id, wh_type, jacket_r)
+                          nbeds, boiler, boiler_plant_loop, boiler_fuel_type, dhw_map, sys_id, wh_type, jacket_r, standby_loss)
     obj_name_indirect = Constants.ObjectNameWaterHeater
     convlim = model.getConvergenceLimits
     convlim.setMinimumPlantIterations(3) # add one more minimum plant iteration to achieve better energy balance across plant loops.
 
     if wh_type == "space-heating boiler with storage tank"
       tank_type = Constants.WaterHeaterTypeTank
+      # Actual tank volume = 95% nominal tank volume
+      act_vol = calc_storage_tank_actual_vol(vol, nil)
+      a_side = calc_tank_areas(act_vol)[1]
+      ua = calc_indirect_tank_ua(act_vol, standby_loss)
+
+      # Tank jacket
+      # assume indirect water heater skin to be insulated with 2inch R5 (the same as electric)
+      ua = apply_tank_jacket(jacket_r, nil, Constants.FuelTypeElectric, ua, a_side)
```

Comment:
Can we move this method call inside of calc_indirect_tank_ua? Then it will be consistent with calc_tank_ua.

Reply:
If doing so, ERI eec calculation will call this function with wrap applied. It's probably harder to revert jacket changes in ERI.

Reply:
Wouldn't we be able to pass in jacket_r = 0 when calc_indirect_tank_ua is called from ERI?

Reply:
Oh true. I will address this.